gulpfile.js

```diff
@@ -132,40 +141,34 @@ function sassTask(source) {
 //——————————————————————————————————————————————————————————————————————————————
 // JS Lint + Bundle
 //——————————————————————————————————————————————————————————————————————————————
-// gulp.task('dev:js', ['dev:js-lint', 'dev:js-bundle']);
-
-
-//——————————————————————————————————————————————————————————————————————————————
-// Build assets and start development server
-//——————————————————————————————————————————————————————————————————————————————
-gulp.task('dev', ['dev:sass', /*'dev:js',*/ 'dev:bs', 'dev:jekyll', 'watch']);
+// function 'dev:js', ['dev:js-lint', 'dev:js-bundle']);
 
 
 //——————————————————————————————————————————————————————————————————————————————
 // Watch Files For Changes
 //——————————————————————————————————————————————————————————————————————————————
-gulp.task('watch', () => {
+function watchTask() {
   // gulp.watch('_js/**/*.js', ['dev:js']);
-  gulp.watch(['common-design/**/*.scss'], ['dev:sass:commondesign']);
-  gulp.watch(['ocha/**/*.scss'], ['dev:sass:ochaextras']);
-  gulp.watch(['styleguide/**/*.scss'], ['dev:sass:styleguide']);
-  gulp.watch(['_config*', '**/*.{md,html,json}', '!_site/**/*.*', '!node_modules/**/*.*'], ['dev:jekyll']);
-});
-
-
-//——————————————————————————————————————————————————————————————————————————————
-// Build site for a deploy to production
-//——————————————————————————————————————————————————————————————————————————————
-gulp.task('deploy', ['dev:sass', /*'dev:js'*/]);
+  gulp.watch(['common-design/**/*.scss'], sassCommonDesign);
+  gulp.watch(['ocha/**/*.scss'], sassOchaExtras);
+  gulp.watch(['styleguide/**/*.scss'], sassStyleguide);
+  gulp.watch(['_config*', '**/*.{md,html,json}', '!_site/**/*.*', '!node_modules/**/*.*'], jekyllTask);
+};
+exports.watch = watchTask;
 
 
 //——————————————————————————————————————————————————————————————————————————————
-// Offer help on command line
+// DEFAULT
+// Build assets and start development server
 //——————————————————————————————————————————————————————————————————————————————
-gulp.task('help', taskListing);
+const defaultTask = gulp.parallel(gulp.series(sassAll, jekyllTask, browserSyncTask), watchTask);
+exports.default = defaultTask;
+exports.dev = defaultTask;
 
 
 //——————————————————————————————————————————————————————————————————————————————
-// Help task is default
+// Build site for a deploy to production
+//
+// @see .bin/deploy.sh
 //——————————————————————————————————————————————————————————————————————————————
-gulp.task('default', ['help']);
+exports.deploy = gulp.series(sassAll);
```

Comment:
@left23 just FYI, here's an example of why I wanted to leave the task names the same. This gulp task is used by `.bin/deploy.sh` to compile in production mode. Leaving publicly-defined tasks as the same name reduces other maintenance sads from happening.